docs/source/platform_technical_documentation/custom-environments.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
<ol><li>Can we include an example yaml in this repo?</li><li>Also can we show how to use this yaml to install to the <code>base</code>,installing outside of <code>base</code> is only necessary when conflicts occur.</li><li>We need another section showing how to select the correct kernel in a notebook, and that these commands could be run from a notebook too.</li></ol><p> </p><p>"Some python packages are not available via <code>mamba</code>." <code>conda-forge</code> it's critical to be clear that's where packages come from.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks, agreed, will make these changes tomorrow. </p>

Reply:
<p>still need kernel selection inside a notebook screenshot</p>

Reply:
>   Can we include an example yaml in this repo?
>     Also can we show how to use this yaml to install to the base,installing outside of base is only necessary when conflicts occur.
>     We need another section showing how to select the correct kernel in a notebook, and that these commands could be run from a notebook too.
> 
> "Some python packages are not available via mamba." conda-forge it's critical to be clear that's where packages come from.

Done